src/app/shared/cookies/browser-klaro.service.ts

```diff
@@ -257,4 +298,11 @@ export class BrowserKlaroService extends KlaroService {
   getStorageName(identifier: string) {
     return 'klaro-' + identifier;
   }
+
+  /**
+   * remove the google analytics from the services
+   */
+  private filterConfigServices(servicesToHide: Pick<typeof klaroConfiguration, 'name'>[]): Pick<typeof klaroConfiguration, 'services'>[] {
+    return this.klaroConfig.services.filter(service => !servicesToHide.some(el => el.name === service.name));
```

Comment:
A mentioned above: if this method is the only reason to use the `Pick<...>` type, please refactor it to work with just the names as a `string[]`.

Reply:
As above:
> The reason is to clearify that we are expecting the same property type as the one named name in the klaroConfiguration object, in this way we can easily compare them using the === (triple equal)